OpenRA.Game/Widgets/Widget.cs

```diff
@@ -169,7 +178,10 @@ public class ChromeLogic : IDisposable
 
 	public abstract class Widget
 	{
-		public readonly List<Widget> Children = new List<Widget>();
+		public readonly YogaNode Node = new YogaNode();
```

Comment:
Wouldn't it be better to encapsulate this inside the Widget class?
Right now we have cases of `Root.Node.Width =` etc https://medium.com/better-programming/demeters-law-don-t-talk-to-strangers-87bb4af11694
Or is it planned for followup pr?

Reply:
Its a temporary workaround to support both layout variants. The old approach has a string set which includes variables and is used by the fieldloader. The yoga variant uses different values here. Idea was to first support two completely different layout mechanisms at the same time, and after a given time, we drop the old implementation and make Widget properly inherit YogaNode.